### DISCUSSION TITLE
Connection pool limit

### DISCUSSION THREAD

**Discussion by ezorita** [RobertCraigie/prisma-client-py#1043] [Q&A]
I am sharing a prisma postgresql connection among many threads and eventually hitting the connection pool limit.
```
prisma.errors.DataError: Timed out fetching a new connection from the connection pool. More info: http://pris.ly/d/connection-pool (Current connection pool timeout: 10, connection limit: 5) 
```
Prisma docs suggest to either append `?connection_limit=` to the database URL or pass an argument to the `prisma.connect()` method, but neither of them seem to be supported in prisma-python.

I see that the Prisma class constructor allows to set the `http.limits` but the defaults are 1000 connection and does not seem to match what prisma is reporting.

Any help will be appreciated, thanks!

**Comment by steinathan** [comment]
@ezorita is this a supabase connection URL?

**Reply by ezorita** [reply]
It is a PostgreSQL instance.
